code/game/jobs/job/civilians/other/survivors.dm

```diff
@@ -7,11 +7,12 @@
 	total_positions = 8
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT|ROLE_CUSTOM_SPAWN
 	late_joinable = FALSE
+	var/is_synth = FALSE
```

Comment:
Missed 10,27, 32-39 on removing it, the only thing you need in this survivors.dm is the ratio changes, the rest is synth related as said before